cmd/filestransfer/filestransfer.go

```diff
@@ -0,0 +1,180 @@
+// Example: Files transfer using PUT/GET commands
+//
+// This example shows how to transfer files to staging area, from which data can be loaded into snowflake
+// database tables. Apart from sending files to staging area using PUT command, files can also be downloaded
+// using GET command.
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	sf "github.com/snowflakedb/gosnowflake"
+)
+
+const customFormatCsvDataToUpload = "NUM; TEXT\n1; foo\n2; bar\n3; baz"
```

Comment:
Go supports multiline using backticks. Maybe it would be more readable?

Reply:
I tried it, but unfortunately it wasn't due to gomt formatting. First row wasn't aligned with the bottom rows and it looked weird.